src/loss_posteriori.jl

```diff
@@ -254,9 +173,10 @@ function loss_MulDtO_oneset(trajectory,
     list_starts = cat([trajectory[:, :, i] for i in starting_points]...,
         dims = 2)
     # Use the differentiable solver to get the predictions
-    pred, list_tr = predict_u_CNODE(list_starts, θ, st, training_CNODE, list_tr)
+    pred, target = predict_u_CNODE(list_starts, θ, st, nunroll, training_CNODE, list_tr)
     # the loss is the sum of the differences between the real trajectory and the predicted one
-    loss = sum(abs2, list_tr .- pred) ./ sum(abs2, list_tr)
+    loss = sum(abs2, target[:, :, 2:nunroll] .- pred[:, :, 2:nunroll]) ./
```

Comment:
@SCiarella I am having some problems with the dimensions of vectors when calculating the loss, I do not understand why are you getting rid of the first time step? Is this considering that there is a fix overlap of 1?

Reply:
the first step is the initial condition for the NODE, so by construction `target[:, :, 1] == pred[:, :, 1]`